tests/unittests/GraphOptzTest.cpp

```diff
@@ -5451,6 +5451,31 @@ TEST_F(GraphOptz, EliminateSliceConcatTest) {
   checkNumericalEquivalence();
 }
 
+TEST_F(GraphOptz, EliminateSliceConcatWithReshapeTest) {
+  auto *src =
+      mod_.createPlaceholder(ElemKind::FloatTy, {4, 5, 3}, "src", false);
+  auto *A = F_->createSlice("A", src, {0, 0, 0}, {1, 5, 3});
+  auto *B = F_->createSlice("B", src, {1, 0, 0}, {2, 5, 3});
+  auto *C = F_->createSlice("C", src, {2, 0, 0}, {3, 5, 3});
```

Comment:
As mentioned above:
```suggestion
  auto *src =
      mod_.createPlaceholder(ElemKind::FloatTy, {8, 5, 3}, "src", false);
  auto *A = F_->createSlice("A", src, {0, 0, 0}, {2, 5, 3});
  auto *B = F_->createSlice("B", src, {2, 0, 0}, {4, 5, 3});
  auto *C = F_->createSlice("C", src, {4, 0, 0}, {6, 5, 3});
```

Reply:
Explained in a previous comment. Only works if the slice size along the consecutive dimension is 1.